packages/mobile/src/components/Modal.tsx

```diff
@@ -0,0 +1,29 @@
+import Card from '@celo/react-components/components/Card'
+import colors from '@celo/react-components/styles/colors.v2'
+import * as React from 'react'
+import { StyleProp, StyleSheet, ViewStyle } from 'react-native'
+import ReactNativeModal from 'react-native-modal'
+
+interface Props {
+  children: React.ReactNode
+  isVisible: boolean
+  style?: StyleProp<ViewStyle>
+}
+
+export default function Modal({ children, isVisible, style }: Props) {
+  return (
+    <ReactNativeModal isVisible={isVisible} backdropOpacity={0.1}>
+      <Card style={[styles.root, style]} rounded={true}>
```

Comment:
We need to wrap the `Card` inside `<SafeAreaView>` so it doesn't render below the status bar or under the home indicator on iOS.

Reply:
done